docs/dashboard/monitoring/README.md

```diff
@@ -0,0 +1,169 @@
+<!--
+title: Serverless Dashboard - Monitoring
+menuText: Monitoring
+menuOrder: 1
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/dashboard/insights/)
```

Comment:
Is monitoring a part of insights? Or is monitoring another first-class citizen (in that case we might want to update the link)?